src/src/hw9/hillel/ArrayDataException.java

```diff
@@ -0,0 +1,7 @@
+package hw9.hillel;
+
+public class ArrayDataException extends Exception {
```

Comment:
Extending from `Exception` is not really correct. Would be enough to extend from `RuntimeException`